runner/src/internal.rs

```diff
@@ -34,24 +34,11 @@ pub struct Opt {
 
 #[derive(StructOpt, Clone)]
 pub enum Command {
-    BuildExamples(BuildExamples),
     RunExamples(RunExamples),
     BuildServer(BuildServer),
+    Format,
     RunTests,
-}
-
-#[derive(StructOpt, Clone)]
-pub struct BuildExamples {
-    #[structopt(long, help = "package application in a Docker image")]
-    docker: bool,
```

Comment:
Do we no longer need this option? It seems like it wasn't implemented either way, but am curious if we want it